operator/flags.go

```diff
@@ -315,5 +315,8 @@ func init() {
 	flags.String(option.BGPConfigPath, "/var/lib/cilium/bgp/config.yaml", "Path to file containing the BGP configuration")
 	option.BindEnv(option.BGPConfigPath)
 
+	flags.Bool(option.SkipCRDCreation, false, "Skip Kubernetes Custom Resource Definitions creations")
```

Comment:
nit: I would reword the help message in a way that differ from what can be guessed by the flag name itself:
```suggestion
	flags.Bool(option.SkipCRDCreation, false, "when true, Kubernetes Custom Resource Definitions will not be created")
```